Dockerfile

```diff
@@ -58,10 +58,11 @@ RUN wget https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/p
 
 # Install ord-schema.
 WORKDIR ..
-# Bust the cache to get the latest commits; see https://stackoverflow.com/a/39278224.
-ADD "https://api.github.com/repos/Open-Reaction-Database/ord-schema/git/refs/heads/main" ord-schema-version.json
 RUN git clone https://github.com/Open-Reaction-Database/ord-schema.git
 WORKDIR ord-schema
+# NOTE(kearnes): ord-schema is versioned to avoid breaking ord-editor.
+ARG ORD_SCHEMA_SHA=79d330952a1d82c5061f37c2d3d5859213507487
```

Comment:
How is this going to stay current? Is someone promising to update it from time to time? Is there a release process?

Reply:
Yes, it will need to be updated manually. This will be ad hoc by us until we need something more formal.

Once we start making tagged releases for ord-schema we can use those tags instead of SHAs.

Reply:
Is it too early to start doing tagged releases?

Reply:
I think we should make a tag for this now so we don't need a PR on ord-editor just to do the ord-schema release.

Reply:
Just chatted with Anton offline: using a tag would be more readable, and we could use a moving tag if we wanted.

For the purpose of being explicit, I think unique tags on ord-schema that we update with PRs to ord-editor makes the most sense. I'll take responsibility for making sure that ord-editor is updated regularly and especially when changes are made to the protos.